frontend/src/components/HacknightsParticipants.vue

```diff
@@ -89,6 +93,11 @@ export default {
       this.$store
         .dispatch('hacknight/addParticipants', ids)
         .then(() => (this.selectedParticipants = []));
+    },
+    orderedParticipants() {
+      return _.sortBy(this.getHacknight.participants, participant =>
```

Comment:
Any reason why `localeCompare` wouldn't handle this? Seems like we could even have one helper function defined in `methods`, that would handle sorting and then use it for both places where it's needed in this component. That would be more DRY solution.

Reply:
In my opinion sortBy is more descriptive and easier to follow, it is also more stable sorting algorithm and in most cases faster. In one method we want to order participants, in another filter them out, so for me at least, these are different functionalities, and I would prefer to keep them separately.

Reply:
I'm just asking because you've used `localeCompare` above. In current version it's line 96. Methods you're talking about are different indeed but they have the common sorting part, which I'm suggesting to extract.